### PR TITLE
Meta metatile

### PR DESCRIPTION
**Pull request by dnomadb**
@perrygeo the functionality we've been discussing is here

### CODE REVIEW

File: bin/colonel-mercator [hunk 1]
```diff
@@ -2,41 +2,90 @@
...
-});
\ No newline at end of file
+    colonel_mercator.drill_raster(filepath, maxres, snap, function(err, pixel) {
+      if (err) returnError(err);
+      console.log(pixel[0])
+    });
+
+} else if (subcommand === 'metatile') {
+    if (argv._.length != 2) {
+        returnError('Usage: colonel-mercator resolution <res> <opts>');
```
Comment (perrygeo) [inline bin/colonel-mercator]: `s/resolution/metatile/` ?

Comment (dnomadb) [inline bin/colonel-mercator]: @perrygeo yes! twas `copypasta`

**Comment by perrygeo** [comment]
I'm getting weird results on an `epsg:4326` file

```
$ rio info --res ~/data/srtm/srtm_med.tif
0.016666666664 0.016666666664
```

So 1 arc-second or ~ 2000 meters res.

```
$ colonel-mercator resolution ~/data/srtm/srtm_med.tif
206533645.15728384
```

That's a big pixel. If I manually warp the image to 3857, things look fine.

**Comment by perrygeo** [comment]
working now

```
$ colonel-mercator resolution ~/data/srtm/srtm_med.tif
1855.3240737772223
$ colonel-mercator resolution ~/data/srtm/srtm_med.tif --snap
2445.98388671875
```

**Comment by dnomadb** [comment]
Cause: didn't realize that the digested cellsize was already converted to `m`, so there was no need to convert it again. Added a testcase for 4326 imagery.

**Pull request merged**